torchbenchmark/models/demucs/demucs/utils.py

```diff
@@ -177,7 +177,7 @@ def save_model(model, path):
     th.save((klass, args, kwargs, state), save_to)
 
 
-def capture_init(init):
+def capture_init(init: Callable) -> Callable:
```

Comment:
Are we supporting `Callable` now? If so, need to add to the language spec (of course, not to be addressed in this issue)